lib/soap/response.ex

```diff
@@ -7,4 +11,13 @@ defmodule Soap.Response do
           request_url: String.t(),
           status_code: pos_integer()
         }
+
+  alias Soap.Response.Parser
+
+  @doc """
+  Executing with xml response body as string.
+  If body is an empty then parse/1 returns a full parsed response structure as map.
```

Comment:
in this place you use parse/1

Reply:
what does it mean a full parsed response structure if the body is empty?

Reply:
Done